src/Analyzer.JsonRuleEngine.UnitTests/LeafExpressionTests.cs

```diff
@@ -49,29 +43,37 @@ public void Evaluate_ValidScope_ReturnsResultsOfOperatorEvaluation(string resour
             };
 
             // JObject to evaluate, in this test, this is a subset of an ARM template
-            var jsonToEvaluate = JObject.Parse(jtoken);
+            var jsonToEvaluate = JObject.Parse("{ \"property\": \"value\" }");
 
-            // Setting up the Mock JsonPathResolver to return the expected values when JToken and Resolve are called
+            // Setting up the Mock JsonPathResolvers to return the expected values when JToken and Resolve are called
             var mockJsonPathResolver = new Mock<IJsonPathResolver>();
+            var mockResourcesResolved = new Mock<IJsonPathResolver>();
```

Comment:
*mockResourceResolver

Reply:
I chose the name `mockResourcesResolved` because it's not a "resource resolver", it's a mock of a resolved resource.  Is there a better name I can use to clarify this?